armory/utils/config_schema.json

```diff
@@ -87,8 +87,6 @@
             },
             "required": [
                 "kwargs",
-                "module",
-                "name",
```

Comment:
Looking at this again, I think that it might be better to leave these as required fields, and simply set them to `null` in the PerfectFiltering configs.

Reply:
There's some regex matching that doesn't allow `null`.  We could rewrite to allow null as an option?

Reply:
Can you try the empty string `""` instead of `null`?

Reply:
It does not accept the empty string either

Reply:
That is weird. I guess do `"null"`. That should work. Technically, it could be a module and a name. :)

Reply:
Check out the latest commit, where config validation allows `null` for module and name (although this affects the module field anywhere, not just under defense).  What are your thoughts?

Reply:
Oops, your last message didn't show up before I posted the above

Reply:
I'll switch it back and use `"null"`

Reply:
Yeah, let's use `"null"`. The modification of the type checking will have broader impact than I would like.